src/VbsEnclaveSDK/src/veil_host_lib/utils.vtl0.h

```diff
@@ -3,9 +3,33 @@
 
 #pragma once
 
+#include "pch.h"
 #include <mutex>
+#include <winrt/Windows.Security.Credentials.h>
+#include <winrt/Windows.Storage.Streams.h>
 
 namespace veil::vtl0::implementation
```

Comment:
>implementation [](http://example.com/codeflow?start=22&length=14)

We use implementation everywhere else, but i think i like `internal` much better.  Let's keep it.  Maybe we move everything to 'internal'.